beacon-chain/types/block.go

```diff
@@ -32,14 +31,10 @@ func NewBlock(data *pb.BeaconBlock) (*Block, error) {
 				ActiveStateHash:       []byte{0},
 				CrystallizedStateHash: []byte{0},
 			},
-		}, nil
+		}
 	}
 
-	if len(data.ParentHash) != 32 {
```

Comment:
Is there a reason why this was removed? I asked @s1na to add this check because protobufs can't validate that a byte array is a fixed length, and in practice a malicious peer can send a byte array of any length.

Reply:
This occurs because protobufs cannot store [32]byte but only []byte, so it is not possible for this condition to be met